index.js

```diff
@@ -135,14 +135,13 @@ module.exports = function(tilelive, options) {
               return next(err);
             }
 
-            return async.waterfall([
-              async.apply(im.encode, "webp"),
-              function(img, done) {
-                vt.addImage(img, src.info.id.replace(/\./g, "_"));
```

Comment:
Oh, n/m. This is what the error is related to. The fix might be to bind the image to `encode`:

```javascript
async.apply(im.encode.bind(im), "webp"),
```